Code/Framework/AzCore/AzCore/Memory/SystemAllocator.cpp

```diff
@@ -15,140 +15,54 @@
 #include <AzCore/std/functional.h>
 
 #include <AzCore/Debug/Profiler.h>
+#include <iostream>
```

Comment:
This include is unused and should be removed.

Reply:
Fixed.